amy/consents/tests/test_forms.py

```diff
@@ -1,11 +1,12 @@
-from consents.forms import ActiveTermConsentsForm
+from consents.forms import ActiveTermConsentsForm, RequiredConsentsForm
 from consents.models import Consent, Term, TermOption
+from django.db.models.fields import BLANK_CHOICE_DASH
 from django.test import TestCase
 from django.utils import timezone
 from workshops.models import Person
 
 
-class TestTermForm(TestCase):
+class TestActiveTermConsentsForm(TestCase):
```

Comment:
You introduced a new method to these forms, are you testing it?

Reply:
@pbanaszkiewicz what new method are you referring to? The only additional method is the `get_terms` method that is called in the init function. Is that what you mean? That is covered by initializing the form

Reply:
Yes, I meant `get_terms`. It's okay if there are tests for it.